src/common/gui/base/Dropdown.ts

```diff
@@ -162,9 +159,8 @@ export class Dropdown implements ModalComponent {
 			// We need to set the height to the height of the parent which already has the calculated and measured height, otherwise this element might
 			// overflow the parent (the overall dropdown container) when there's not enough vertical space to display all items
 			return m(
-				".dropdown-content.scroll.height-100p",
+				".dropdown-content.scroll",
```

Comment:
we added `.height-100p` with a previous fix for #7937. did you test that it didn't break again?